app/src/main/java/com/duycuong/weather/ui/screen/main/MainActivity.java

```diff
@@ -4,12 +4,14 @@
 import android.annotation.SuppressLint;
 import android.content.pm.PackageManager;
 import android.databinding.DataBindingUtil;
+import android.drm.DrmStore;
```

Comment:
ERROR: Unused import: android.drm.DrmStore. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports